hardware_interface/test/test_component_interfaces.cpp

```diff
@@ -43,7 +43,7 @@ namespace hardware_interfaces_components_test
 class DummyPositionJoint : public components::Joint
```

Comment:
as mentioned in #171, having a base class within this package doesn't look like the right approach.

Reply:
Agree, we can move this test with #171